src/main/java/bio/terra/service/profile/ProfileService.java

```diff
@@ -77,12 +78,16 @@ public ProfileService(
    * @return jobId of the submitted stairway job
    */
   public String createProfile(
-      BillingProfileRequestModel billingProfileRequest, AuthenticatedUserRequest user) {
+      BillingProfileRequestModel billingProfileRequest,
+      AuthenticatedUserRequest user,
+      Optional<String> idpAccessToken) {
     String description =
         String.format("Create billing profile '%s'", billingProfileRequest.getProfileName());
-    return jobService
-        .newJob(description, ProfileCreateFlight.class, billingProfileRequest, user)
-        .submit();
+    var jobBuilder =
+        jobService.newJob(description, ProfileCreateFlight.class, billingProfileRequest, user);
+    idpAccessToken.ifPresent(
```

Comment:
So, I like this pattern but from a consistency standpoint, we've avoided passing optionals and done explicit null checks for operations like this

Reply:
so you want me to change all the optionals to nullables?

Reply:
I mean, don't get me wrong, I prefer this pattern but for consistency yeah I'd pass in nullables

Reply:
I am going to push back a tiny bit but if you insist, I will cave. IMO, it is more than style, `Optional` signifies intent.

Reply:
Nah, I don't really insist and I agree with you.  I'm happy to start moving more in this direction.  I don't know if anyone else wants to chime in here but I say go for it, but I want to make sure that we actually move in this direction then moving forward